tests/Tests/500__jupyterhub/test-versions.robot

```diff
@@ -94,3 +105,17 @@ Load Spawner Page
     [Documentation]    Suite Setup, loads JH Spawner
     Begin Web Test
     Launch JupyterHub Spawner From Dashboard
+
+Verify Python Version In Notebook
```

Comment:
There is already a `Python Version Check` keyword in JupyterLabLauncher.robot

Reply:
But it checks only if version == given version, Here I am required to check version>=3.8

Reply:
you could create a keyword `Get Python Version From Notebook`, move the code lines from `Python Version Check` before the `Should Match` keyword to the new keyword. The you could either:
1. rename `Python Version Check`  to `Python Version Should Be Equal` and create a new keyword `Python Version Should Be GTE` 
2. You change `Python Version Check`  to `Python Version Should Be` to take as argument the operator (e.g., equal, gte, lte etc) and make it a generic keyword to check version

Reply:
My suggestion last Friday was to talk with @lugi0 to see the best way to check this, as Luca is already checking Python and other stuff for all notebook images